app/controllers/api/v0/api_controller.rb

```diff
@@ -91,6 +92,11 @@ def add_missing_api_key_warning_header
         # rubocop:enable Layout/LineLength
       end
 
+      def api_usage_tracking
+        usage_tags = ["endpoint:#{request.path}", "user_id:#{@user&.id || 'unauthenticated'}"]
```

Comment:
I was thinking we could also add something to identify which HTTP verb was used, but I'm unsure if it would be better to use a single tag to identify method+endpoint (i.e. `GET /api/comments`):

```suggestion
        usage_tags = ["endpoint:#{request.method} #{request.path}", "user_id:#{@user&.id || 'unauthenticated'}"]
```

or in a separate tag:

```suggestion
        usage_tags = ["endpoint:#{request.path}", "user_id:#{@user&.id || 'unauthenticated'}", "method:#{request.method}"]
```